.github/workflows/periodic_release.yml

```diff
@@ -0,0 +1,59 @@
+name: Periodic Release
+
+on:
+  schedule:
+    # Every Monday at 5 PM UTC (9 AM PST)
+    - cron: "0 17 * * 1"
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.daemon=false -Xmx2g -Xms2g
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle.kts') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Write current version
+        run: |
+          export CURRENT_TAG=`curl --silent "https://api.github.com/repos/$1/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'`
+          echo "version: $CURRENT_TAG" > package.json
```

Comment:
That's a hack, but the best I could come up with to make this work without this off-the-shelf action, which requires the current version to be written to a file.